index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <link rel="stylesheet" href="./src/style">
+</head>
+<body>
+<div class="gallery">
+    <div class="main">
+            <img src="https://js.cx/gallery/img1-lg.jpg" alt="Large image" class="large">
+    </div>
+    <div class="bottom-img">
```

Comment:
When I read it in js I think about 1 image, better to name `images`, `images-list`, `images-container` etc.